storage/storage_client.go

```diff
@@ -101,7 +102,7 @@ func (c *StorageClient) getUnqualifiedName(name string) string {
 	}
 
 	nameParts := strings.Split(name, "/")
-	return strings.Join(nameParts[2:], "/")
+	return strings.Join(nameParts[len(nameParts)-1:], "/")
```

Comment:
And I think this line https://github.com/hashicorp/go-oracle-terraform/blob/96994b53ceb054c8a5bb321d7139748fe1b20079/storage/storage_client.go#L100 protects us since it'll always be greater than 2 or we wouldn't get this far